ipaserver/install/ldapupdate.py

```diff
@@ -328,12 +329,15 @@ def _identify_arch(self):
            etc.  Determine if a suffix is needed based on the current
            architecture.
         """
-        bits = platform.architecture()[0]
-
-        if bits == "64bit":
-            return "64"
+        if NAME is 'debian':
+            return paths.LIBARCH
         else:
-            return ""
+            bits = platform.architecture()[0]
```

Comment:
Please use ``sys.maxsize`` here: https://docs.python.org/3/library/sys.html#sys.maxsize

Reply:
I'd really rather go with my proposal above and get all platform/arch specific code to `ipaplatform`

Reply:
That's a good idea, too.